api/ca.proto

```diff
@@ -22,6 +22,11 @@ service NodeCA {
 	rpc NodeCertificateStatus(NodeCertificateStatusRequest) returns (NodeCertificateStatusResponse) {
 		option (docker.protobuf.plugin.tls_authorization) = { insecure: true };
 	};
+	// GetUnlockKey returns the current unlock key for the cluster for the role of the client
+	// asking.
+	rpc GetUnlockKey(GetUnlockKeyRequest) returns (GetUnlockKeyResponse) {
```

Comment:
Agreed. My thoughts are: `CA` hosts all the shared methods (that might be useful for the future `ServiceCA`), and `NodeCA` only hosts Node specific certificate actions.